GHA: `optimize` runs failing to install cyipopt

good: https://github.com/ICB-DCM/pyPESTO/actions/runs/13171421231/job/36762358599
bad: https://github.com/ICB-DCM/pyPESTO/actions/runs/13299992254/job/37139587934

```
....
      /usr/bin/ld: cannot find -llapack: No such file or directory
      /usr/bin/ld: cannot find -lblas: No such file or directory
...
```

Probably due to updated runner images, although not evident from the changes: 
https://github.com/actions/runner-images/compare/releases/ubuntu24/20250202...releases/ubuntu24/20250209

--- a/.github/workflows/install_deps.sh
+++ b/.github/workflows/install_deps.sh
@@ -42,6 +42,12 @@ for par in "$@"; do
 	build-essential \
         coinor-libipopt1v5 coinor-libipopt-dev \
         gfortran lcov pkg-config python3-dev zlib1g-dev
+      if [ ! -f /usr/lib/x86_64-linux-gnu/liblapack.so ]; then
+        sudo ln -s /usr/lib/x86_64-linux-gnu/liblapack.so.3 /usr/lib/x86_64-linux-gnu/liblapack.so
+      fi
+      if [ ! -f /usr/lib/x86_64-linux-gnu/libblas.so ]; then
+        sudo ln -s /usr/lib/x86_64-linux-gnu/libblas.so.3 /usr/lib/x86_64-linux-gnu/libblas.so
+      fi
     ;;
 
     pysb)

